bundles/org.openhab.binding.anthem/README.md

```diff
@@ -0,0 +1,83 @@
+# Anthem Binding
+
+The binding allows control of Anthem AV processors over an IP connection to the processor.
+
+## Supported Things
+
+The following thing type is supported:
+
+| Thing    | ID       | Discovery | Description |
+|----------|----------|-----------|-------------|
+| Anthem   | anthem   | Manual | Represents a Anthem AV processor |
+
+Tested models include the AVM-60 11.2-channel preamp/processor.
+
+## Discovery
+
+There is no automatic discovery.
+Things must be added manually using the user interface or a `.things` file.
+
+## Thing Configuration
+
+The following configuration parameters are available on the Anthem thing:
+
+| Parameter           | Parameter ID              | Required/Optional | Description |
+|---------------------|---------------------------|-------------------|-------------|
+| Host                | host                      | Required          | IP address or host name of the Anthem AV processor |
+| Port                | port                      | Optional          | Port number used by the Anthem |
+| Reconnect Interval  | reconnectIntervalMinutes  | Optional          | The time to wait between reconnection attempts (in minutes) |
+| Command Delay       | commandDelayMsec          | Optional          | The delay between commands sent to the processor (in milliseconds) |
+
+## Channels
+
+The Anthem AV processor supports the following channels (some zones/channels are model specific):
+
+| Channel                 | Type    | Description  |
+|-------------------------|---------|--------------|
+| *Main Zone*             |         |   |
+| 1#power                 | Switch  | Power the zone on or off  |
+| 1#volume                | Dimmer  | Increase or decrease the volume level  |
+| 1#volumeDB              | Number  | The actual volume setting  |
+| 1#mute                  | Switch  | Mute the volume  |
+| 1#muteToggle            | Switch  | Toggle the mute on and off  |
+| 1#activeInput           | Number  | The currently active input source  |
+| 1#activeInputShortName  | String  | Short friendly name of the active input  |
+| 1#activeInputLongName   | String  | Long friendly name of the active input |
+| *Zone 2*                |         |   |
+| 2#power                 | Switch  | Power the zone on or off  |
+| 2#volume                | Dimmer  | Increase or decrease the volume level  |
+| 2#volumeDB              | Number  | The actual volume setting  |
+| 2#mute                  | Switch  | Mute the volume  |
+| 2#muteToggle            | Switch  | Toggle the mute on and off  |
```

Comment:
What is the difference between these two channels?

Reply:
The Anthem processors support two styles of muting. There's an ON/OFF mute command, and there's a mute toggle command (sending an ON command will change to the opposite state of whatever it's currently set to). I debated whether to just add the mute ON/OFF channel or add two channels, one for ON/OFF and one for toggling the mute. Do you have an opinion?

Reply:
If I understand this correct, they will both toggle, but one of them is stateless? Will the current mute status be reflected in the state of the "normal" mute channel? If yes, then this would also toggle it? If not, they are actually both stateless?

If the latter is the case, do I get this correct?
- Initial processor status: Muted.
- Initial mute status: OFF
- Initial muteToggle status: OFF
- Set mute to ON. Nothing happens, because already muted.
- Set muteToggle to ON: Now will unmute because it's a toggle.
- Both channels are now ON, but volume is actually unmuted?

I'm very confused. 🙂

Reply:
Yeah, my example was incomplete because it was created for the case that they both were stateless. If the mute channel is stateful, I see no need for the toggle channel.

Reply:
muteToggle channel removed